test/vfd_swmr.c

```diff
@@ -3424,6 +3686,701 @@ test_enable_disable_eot(void)
+        TEST_ERROR;
+
+    /* Clean up updater files */
+    for (i = 0; i < seq_num; i++) {
+        HDsprintf(namebuf, "%s.%lu", UD_FILENAME, i);
+        HDremove(namebuf);
+    }
+
+    if (H5Pclose(fcpl) < 0)
+        FAIL_STACK_ERROR;
+
+    /* Free buffers */
+    if (config1)
+        HDfree(config1);
+
+    PASSED();
+    return 0;
+
+error:
+    H5E_BEGIN_TRY
+    {
+        H5Pclose(fapl1);
+        H5Pclose(fcpl);
+        H5Fclose(fid);
+        H5Fclose(fid2);
+    }
+    H5E_END_TRY;
+    if (config1)
+        HDfree(config1);
+    return 1;
+} /* test_updater_flags_same_file_opens() */
+
+/*-------------------------------------------------------------------------
+ * Function:    clean_chk_ud_files()
+ *
+ * Purpose:     This is the helper routine used to clean up
+ *              the checksum file and the updater files.
+ *
+ * Return:      void
+ *
+ * Programmer:  Vailin Choi; October 2021
+ *
+ *-------------------------------------------------------------------------
+ */
+static void
+clean_chk_ud_files(char *md_file_path, char *updater_file_path)
+{
+    char     chk_name[1024 + 4];
+    char     ud_name[1024 + 4];
+    uint64_t i;
+
+    /* Name of the checksum file: <md_file_path>.chk */
+    HDsprintf(chk_name, "%s.chk", md_file_path);
+
+    /* Remove the checksum file if exists.
+       If not, the callback will just continue appending
+       checksums to the existing file */
+    if (HDaccess(chk_name, F_OK) == 0) {
+        HDremove(chk_name);
+    }
+
+    /* Remove all the updater files if exist: <updater_file_path>.<i> */
+    for (i = 0;; i++) {
+        sprintf(ud_name, "%s.%lu", updater_file_path, i);
+        if (HDaccess(ud_name, F_OK) != 0)
+            break;
+        HDremove(ud_name);
+    }
+
+} /* clean_chk_ud_files() */
+
+/*-------------------------------------------------------------------------
+ * Function:    verify_ud_chk()
+ *
+ * Purpose:     This is the helper routine used by
+ *              test_updater_generate_md_checksums() to verify
+ *              contents of the checksum file and the updater files.
+ *              --verify the sequence number in each updater's file header
+ *                corresponds to the ith sequence number of the updater
+ *                file name.
+ *              --verify the tick number in each updater's file header
+ *                corresponds to the tick number stored in the checksum file
+ *              --verify the change_list_len in each updater's file header
+ *                is consistent with num_change_list_entries in each updater's
+ *                change list header
+ *
+ * Return:      0 if test is sucessful
+ *              1 if test fails
+ *
+ * Programmer:  Vailin Choi; October 2021
+ *
+ *-------------------------------------------------------------------------
+ */
+static herr_t
+verify_ud_chk(char *md_file_path, char *ud_file_path)
+{
+    char     chk_name[1024 + 4];      /* Checksum file name */
```

Comment:
Minor: Should we use a constant to replace 1024 here? Also should the local variables be initialized? Should the ";" in 'FAIL_STACK_ERROR; ' and 'TEST_ERROR;' be removed or make them consistent?

Reply:
Fix the constant for chk_name[].
As for the ";" after FAIL_STACK_ERROR and TEST_ERROR, I will add ";" to be consistent with most of those calls in this file.  I checked the files in the test directory, some of those calls have ";" and some without.